modules/react/popup/lib/PopupCard.tsx

```diff
@@ -64,7 +64,7 @@ export const PopupCard = createSubcomponent('div')({
   }, [model.state.placement]);
 
   // As is a Stack that will render an element of `Element`
-  const As = useConstant(() => Stack.as(Element));
+  const As = useConstant(() => Flex.as(Element));
```

Comment:
I noticed your comment about this in the PR summary:

> We've changed the base element on Popup.Card to use Flex instead of Stack to prevent breaking changes. If you were using spacing on that component, you'll have to use FlexStyleProps instead.

Can you clarify why this change was made? Which breaking changes are you referring to?

Reply:
Just noticed this bit from the [7.4.0 Announcement](https://canvas.workday.com/whats-new/announcements#canvas-kit-740-release):

> As part of the Modal focus ring bug fix, each section (Heading, Body, and Footer) was separated to have their own padding and margins. Because of this change, the Stack in Modal.Card removed the margin top for the first element (Heading) in the Modal. As a result, the underlying Stack element for Modal.Card has also been replaced with the Flex element to add the 24px margin top back to the Modal Heading. Teams using the spacing prop on Modal.Card will need to remove it. If vertical alignment is required, teams should implement a VStack element.

Reply:
Yeah i would love some word smithing for this. What happened was because i moved margin to the heading element, `Stack` has a selector that was overriding this margin and setting it to `marginTop: 0` so I had to change this to use flex stack so that it would avoid this.